packages/casting/src/follower-cosmjs.js

```diff
@@ -334,18 +334,36 @@ export const makeCosmjsFollower = (
         }
       }
       lastBuf = buf;
-      const data = decode(buf);
-      if (!unserializer) {
-        /** @type {T} */
-        const value = data;
-        committer.commit({ value });
-        return;
+      let streamCell = decode(buf);
+      // Upgrade a naked value to a JSON stream cell if necessary.
+      if (!streamCell.height || !streamCell.values) {
+        streamCell = { values: [JSON.stringify(streamCell)] };
       }
-      const value = await E(unserializer).unserialize(data);
-      if (!committer.isValid()) {
-        return;
+      for (let i = 0; i < streamCell.values.length; i += 1) {
+        const data = JSON.parse(streamCell.values[i]);
+        const last = i + 1 === streamCell.values.length;
+        if (!unserializer) {
+          /** @type {T} */
+          const value = data;
+          committer.commit({ value });
+          if (!last) {
+            committer = prepareUpdateInOrder();
+          }
+          // eslint-disable-next-line no-continue
+          continue;
```

Comment:
I wonder whether we should globally disable this rule. If it's a worthwhile rule to follow, consider refactoring so this suppression isn't necessary. E.g,
```js
      streamCells.values.forEach((dataStr, i) => {
        const data = JSON.parse(dataStr);
        const isLast = i + 1 === streamCell.values.length;
```
so that this can be,
```suggestion
          if (!last) {
            committer = prepareUpdateInOrder();
          }
          return;
```